packages/devtools_app/test/test_infra/flutter_test_environment.dart

```diff
@@ -108,13 +106,14 @@ class FlutterTestEnvironment {
 
       _needsSetup = false;
 
-      _flutter = _flutterDriverFactory(Directory(testAppDirectory));
-      await _flutter.run(
+      _flutter = _flutterDriverFactory(Directory(testAppDirectory))
+          as FlutterRunTestDriver?;
```

Comment:
why does this need a nullable cast? _flutterDriverFactory should return a non-nullable value

Reply:
`_flutter` can probably be marked as a late field instead of being marked nullable (assuming `setupEnvironment` is always invoked at some point).

Reply:
it is assigned to null in tearDownEnvironment